mlir/utils/jenkins/Dockerfile

```diff
@@ -96,7 +96,8 @@ RUN echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https:
 # This make sure that we don't accidentally use newer cmake features incompatible with our client.
 # As of 2022-08-17, the rocm/rock images has /opt/conda/bin/cmake at 3.22.1, so we'll match that.
 # Note that the mlir minimum is 3.18.
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get update && rm /usr/share/keyrings/kitware-archive-keyring.gpg && \
```

Comment:
Instead of rm, remove line `90 - 93`?

Reply:
No, they need to be in that order per the `kitware` instructions for how to enable their repository.

Reply:
https://apt.kitware.com/